week2/homework/src/index.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+let fs = require('fs');
+
+let args = process.argv.slice(2);
+
+let command = args[0];
+let todoItem = args[1];
+
+
+
+if(command === 'add') {
+  add(todoItem);
+} else if (command === 'help'){
+  help();
+} else if (command === 'list') {
+  list(todoItem);
+} else if (command === 'remove') {
+  remove(todoItem);
+};
+
+function list(){
+  fs.readFile('./todoList.txt', 'utf8', (error, todoList) => {
+    if (error){
+      console.log(error);
+    } else {
+      console.log(todoList);
+    };
+  });
+};
+
+function add(todoItem){
+  fs.appendFile('./todoList.txt', todoItem + '\n', (error) => {
+    console.error(error);
```

Comment:
You should also check the error and print it only if there is. Also better to report to user the status, like  

    if(error) {
        console.error(error);
    } else {
        console.log('Item is added.');
    }

Like what you did in `remove` function.